src/main/java/org/bitbuckets/drive/DriveSDSSubsystem.java

```diff
@@ -122,12 +127,9 @@ public void teleopPeriodic() {
                     state = DriveFSM.TELEOP_NORMAL;
                     break;
                 }
-                double IMU_Yaw = Math.toRadians(control.getYaw_deg());//Math.toRadians(-350);
-
-                IMU_Yaw = MathUtil.wrap(IMU_Yaw);
 
                 //will add logic later
-                double setpoint = Math.toRadians(90);
+                double setpoint = Math.toRadians(0);
```

Comment:
Add logic for setting the setpoint, the same way as linear and angular velocities can be set